### DISCUSSION TITLE
wiki for documentaion collaboration

### DISCUSSION THREAD

**Discussion by wlwaw** [puppylinux-woof-CE/woof-CE#3650] [Ideas]
Can there be a wiki for use by those wishing to collaborate on proposed  edits/updates to Puppy documentation in /usr/share/doc ?

**Comment by lakshayrohila** [comment]
Currently [this](https://forum.puppylinux.com/viewtopic.php?t=7273) forum thread is open for update of docs. I also have a draft PR open for the same. Most of the files have been updated, just few need to be updated now.

But this all doesn't provide any contributor guide, about html format or cloning from github etc. Yes, wiki would be a great idea.
